index.bs

```diff
@@ -318,6 +339,285 @@ NOTE: The size of the last layer can be determined by subtracting the sum of the
+    </tr>
+    <tr>
+      <td>9</td>
+      <td>or operator</td>
+      <td>2</td>
+      <td>Bitwise inclusive disjunction of the operands.</td>
+      <td><math><mi>L</mi><mo>∨</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>xor operator</td>
+      <td>2</td>
+      <td>Bitwise exclusive disjunction of the operands.</td>
+      <td><math><mi>L</mi><mo>⊻</mo><mi>R</mi></math></td>
+    </tr>
+    <tr>
+      <td>11</td>
+      <td>not operator</td>
+      <td>1</td>
+      <td>Bitwise complement of the operand.</td>
+      <td><math><mo>¬</mo><mi>L</mi></math></td>
+    </tr>
+    <tr>
+      <td>12</td>
+      <td>bsr operator</td>
+      <td>1</td>
+      <td>0-based index of the most significant set bit of the left operand if the left operand is strictly positive, zero otherwise.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mn>0</mn></mtd><mtd><mo>if</mo><mi>L</mi><mo>≤</mo><mn>0</mn></mtd></mtr>
+            <mtr><mtd>truncate<mo>(</mo><msub><mo>log</mo><mn>2</mn></msub><mi>L</mi><mo>)</mo></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>13</td>
+      <td>pow operator</td>
+      <td>2</td>
+      <td>Left operand raised to the exponent being the right operand if the left operand is not zero, zero otherwise.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mn>0</mn></mtd><mtd><mo>if</mo><mi>L</mi><mo>=</mo><mn>0</mn></mtd></mtr>
+            <mtr><mtd>truncate<mo>(</mo><msup><mi>L</mi><mi>R</mi></msup><mo>)</mo></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>14</td>
+      <td>min operator</td>
+      <td>2</td>
+      <td>Minimum value among the operands.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mi>L</mi></mtd><mtd><mo>if</mo><mi>L</mi><mo>≤</mo><mn>R</mn></mtd></mtr>
+            <mtr><mtd><mi>R</mi></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+    <tr>
+      <td>15</td>
+      <td>max operator</td>
+      <td>2</td>
+      <td>Maximum value among the operands.</td>
+      <td>
+        <math>
+          <mo>{</mo>
+          <mtable>
+            <mtr><mtd><mi>R</mi></mtd><mtd><mo>if</mo><mi>L</mi><mo>≤</mo><mn>R</mn></mtd></mtr>
+            <mtr><mtd><mi>L</mi></mtd><mtd>otherwise</mtd></mtr>
+          </mtable>
+        </math>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<dfn noexport>constant</dfn> is a literal signed value extracted from the stream with a precision of intermediate bit depth, pushed to the stack.
+
+<dfn noexport>input_image_item_index</dfn> is the 1-based index of the input image item whose sample is pushed to the stack. Shall be at most <code>[=reference_count=]</code>.
+
+<h5 id="sample-transform-constraints">Constraints</h5>
```

Comment:
Do we want to have a small section of "common recipes"? Or would that be better to have outside of the spec document itself? (Or potentially in an informative annex)

Reply:
I included the simplest bit depth extension example as a `<div class="example">`. I did not get into the details to keep it short and easy to understand.

The recipes are likely to evolve more often than the specification so I would lean toward another document. Does it have to be officially reviewed or can it be something like a wiki in this repository?

Reply:
> Does it have to be officially reviewed or can it be something like a wiki in this repository?

Let's keep it in the wiki!

Reply:
It would be good to have the two bit depth extension recipies in some spec so they can be standardized, which will enable implementations to handle them as well-known special cases.

Reply:
As discussed, let's have an annex with these patterns.